### ISSUE TITLE
NTR: 4-tyramine present in urine (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8543]
New term request 4-tyramine present in urine (segal_030505193458):

**Comment by pnrobinson** [comment]
@MickeySegal
can you provide context? Is this from exogenous sources?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1515/ "Increased excretion of p-hydroxyphenylpyruvate, p-hydroxyphenyllactate, and p-hydroxyphenylacetate and presence of small quantities of N-acetyltyrosine and 4-tyramine on urine organic acid analysis"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000775
- label: Elevated urinary 4-tyramine level
- parents: [Abnormal urinary nitrogen compound level (HP:0040416)](https://hpo.jax.org/app/browse/term/HP:0040416)
- pmid: PMID:20301688
- def: The amount of 4-tyramine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8543
- synonyms:
